workflows/test.nf

```diff
@@ -70,8 +70,8 @@ workflow TEST {
         file(params.input)
     )
     ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
-    // TODO: OPTIONAL, you can use nf-validation plugin to create an input channel from the samplesheet with Channel.fromSamplesheet("input")
-    // See the documentation https://nextflow-io.github.io/nf-validation/samplesheets/fromSamplesheet/
+    // TODO: OPTIONAL, you can use nf-schema plugin to create an input channel from the samplesheet with Channel.fromSamplesheet("input")
+    // See the documentation https://nextflow-io.github.io/nf-schema/samplesheets/fromSamplesheet/
```

Comment:
@ewels this link is no longer valid, not sure if this is something to be consider either in templates potentially including this comment, or in the automatic migration command suggested by https://nextflow-io.github.io/nf-schema/latest/migration_guide/#updating-the-name-and-version-of-the-plugin, or whether an equivalent samplesheet should be served from the updated location

Reply:
The new link is https://nextflow-io.github.io/nf-schema/latest/samplesheets/samplesheetToList/